src/drivers/mct/main/component_mod.F90

```diff
@@ -32,7 +32,7 @@ module component_mod
   use mct_mod   ! mct_ wrappers for mct lib
```

Comment:
I have removed all whitespace in #1766 accepting that will remove the differences in this one.